Homepage and Logout

AlexMao1 opened the pull request:
Added a html file for the homepage. Modified the homepage() function in app.py to redirect users from login page to homepage. Added a home() function to redirect users from homepage to login page after they pressed logout button.

AlexMao1 committed:
Update app.py

AlexMao1 committed:
Add files via upload

AlexMao1 committed:
Delete Home.html

AlexMao1 committed:
Added the web page for home.

AlexMao1 committed:
Small fixes on the format.

shishu314 reviewed (changes requested):
Broken pages, does not logout. Typos

AlexMao1 committed:
Update app.py

AlexMao1 committed:
Update home.html

AlexMao1 committed:
Update home.html

AlexMao1 committed:
Update home.html

AlexMao1 committed:
Update app.py

AlexMao1 committed:
Update index.html

AlexMao1 committed:
Update register.html

AlexMao1 committed:
Update home.html

AlexMao1 commented:
I fixed the typo and added the session['UserName'] to resolve the problem of not logging out properly. And I think it would be better to keep another route in app.py other than homepage() to render the real home screen that the user actually sees, just like the function home() you did in your branch.

shishu314 reviewed (changes requested):
You still have to fix the merge conflicts with master.

AlexMao1 committed:
Update home.html

AlexMao1 committed:
Update app.py

AlexMao1 committed:
Update app.py

AlexMao1 committed:
Merge branch 'master' into AlexMao1-patch-1
